kamiaslam/mindmeta-frontend
# 21-10-2024 - 10:59 AM - family member birth date validation issue fix

**Summary:**

- A fix was applied to the birthdate issue when creating a family member.

**Testing Instructions:**

- Log in with the "Care Home" role.
- Navigate to the "Create Patient" page.
- While adding a family member, ensure that the birthdate validation is functioning correctly.
